docs/statistics.md

```diff
@@ -294,14 +450,16 @@ Missing packets with sequence numbers 8 and 9 will be reported lost with the nex
 The next received packet has sequence number 8. Reorder tolerance value is increased to 2.
 The packet with sequence number 9 is reported lost.
 
+### pktRcvAvgBelatedTime
 
-
-## pktRcvAvgBelatedTime
+TODO: What's this?
```

Comment:
This may need some attention. The intention of this data was to calculate the average time of a uselessly sent packet that has arrived despite that it was already received (or dropped) in order to get oriented how much the sender failed to prevent a potentially belated packet from being sent.

After time, I think in this form this stat may not exactly help in anything, as it tends to have a "negative cummulation" (packets that were retransmitted and accepted do not contribute to this value at all). OTOH it might give you an idea that there's something wrong if this value tends to be quite high and exceeds the latency (that is, it was sent later than 2* after the latency passed).